react/components/EditorContainer/Sidebar/ConfigurationList/ContentEditor/ConditionControls/Scheduler.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { Component } from 'react'
+import { InjectedIntlProps, injectIntl } from 'react-intl'
+import { withRuntimeContext } from 'vtex.render-runtime'
+import { DatePicker, Toggle } from 'vtex.styleguide'
+
+import {
+  getFirstTime,
+  getLastTime,
+  getNextDay,
+  getNextTime,
+  getPreviousTime,
+  isLastTime,
+  isSameDay,
+} from '../../../../../../utils/date'
```

Comment:
`../../../../../../` 😮 
Could this be a `dateUtils` file closer to this component?

Reply:
I usually try to keep the related and somewhat generic utils in a top-level folder, so I put all date utils in there. I wouldn't mind turning them into local utils while they're not being used anywhere else. Would that be better?

Reply:
No problem, it's OK